crates/bevy_text/src/text2d.rs

```diff
@@ -205,6 +205,17 @@ pub fn update_text2d_layout(
                     panic!("Fatal error when processing text: {e}.");
                 }
                 Ok(mut info) => {
+                    if let Some(min_x) = info
```

Comment:
Could you add a comment here for why this is needed?

Reply:
Yep it should have something, you are right.